src/hooks/api/useCurrentGame.ts

```diff
@@ -45,17 +45,6 @@ export function useCurrentGame() {
     ).finally(() => setLoading(false))
   }
 
-  const closeGame = async () => {
```

Comment:
So only the backend closes the game?

Reply:
yes, (or it closes it before the frontend can, since : https://github.com/sopra-fs22-group-36/screw-your-neighbor-server/pull/155